anesthetic/plot.py

```diff
@@ -47,10 +58,85 @@ def _constructor(self):
     def _constructor_expanddim(self):
         return AxesDataFrame
 
+    @staticmethod
+    def axes_series(data, fig, ncol=None, gridspec_kw=None, subplot_spec=None):
+        """Set up subplots for `AxesSeries`."""
+        axes = data.copy()
+        if fig is None:
+            fig = plt.figure()
+        if ncol is None:
+            ncol = int(np.ceil(np.sqrt(axes.index.size)))
+        ncol = ncol
+        nrow = int(np.ceil(axes.index.size / ncol))
+        if gridspec_kw is None:
+            gridspec_kw = {}
+        wspace = gridspec_kw.pop('wspace', 0)
+        if subplot_spec is None:
+            gs = GridSpec(nrow, ncol, wspace=wspace, **gridspec_kw)
+        else:
+            gs = GridSpecFromSubplotSpec(nrow, ncol, wspace=wspace,
+                                         subplot_spec=subplot_spec,
+                                         **gridspec_kw)
+        for p, g in zip(axes.index, gs):
+            axes[p] = ax = fig.add_subplot(g)
+            ax.set_yticks([])
+        return axes
```

Comment:
Into `AxesSeries._constructor`?  `axes_series` takes a bunch of args and kwargs which doesn't fit in a property such as `_constructor`, I think.

Or do you mean `AxesSeries.__init__`? It could go in that, I guess, but I tried not letting the `__init__` get too bulky...